server/src/uuid/model/entity/mod.rs

```diff
@@ -468,14 +468,22 @@ impl Entity {
         let mut fields = payload.input.fields.clone();
 
         if let Some(revision_id) = last_not_trashed_revision {
-            let last_revision_fields: HashMap<String, String> =
+            let mut last_revision_fields: HashMap<String, String> =
                 fetch_all_fields!(revision_id, &mut transaction)
                     .await?
                     .into_iter()
                     .filter(|field| field.field != "changes")
                     .map(|field| (field.field.to_case(Case::Camel), field.value))
                     .collect();
 
+            // FIXME: This is bad design -> let's have a DB migration?!
```

Comment:
> let's have a DB migration?!

At any rate! This cohesive implies a lot of extra code, opening the doors to bugs.
But what do you have in mind?

Reply:
Migration: Each exercise group revision without a `cohesive` property gets one with `cohesive === false`.